main.py

```diff
@@ -170,11 +170,13 @@ def main(thisPath, driverName, testkitBranch, driverRepo):
     password = "pass"
     neo4jServerHostname = "neo4jserver"
     port = 7687
+    ipv6Enabled = False
```

Comment:
I don't think that we will test ipv6 at this level, too costly to have ipv6 as a test matrix variable. If you need this env variable I suggest you move it in to the driver/javascript/* file